monero-sys/src/lib.rs

```diff
@@ -754,7 +1087,7 @@ impl WalletManager {
     pub fn open_or_create_wallet(
         &mut self,
         path: &str,
-        password: Option<&str>,
+        password: Option<&String>,
```

Comment:
this should stay Option<&str> as a lot of types (including String) implement `AsRef<str>` - basically they can be treated as a `&str` (as string slice/view) - but not `AsRef<String>`.